orchagent/nexthopkey.h

```diff
@@ -98,6 +124,73 @@ struct NextHopKey
     {
         return (ip_address.getV4Addr() == 0);
     }
+
+    bool isMplsNextHop() const
+    {
+        return (label_stack.getSize() > 0);
+    }
+
+    std::string parseMplsNextHop(const std::string& str)
```

Comment:
This method has side effects. would be good if you can add a note saying the "outseg_type" and "label_stack" are updated for label nexthops. Would be even better if you make the method 'const' and returns a tuple.

Reply:
@smaheshm I will add a comment.

Reply:
@smaheshm The comment is added.